### ISSUE TITLE
Question about CD and EMD

### ISSUE THREAD

**Issue by gyjung975** [zyh16143998882/IDPT#9]
Hi, Thanks for your great work!

I have a question about CD and EMD.
There are the codes for the CD and EMD, are they used in the experiments?
I didn't see about it in the paper.

**Comment by zyh16143998882** [comment]
Thank you for your attention. CD and EMD are loss functions used in the self-supervised pre-training stage of point cloud, while our method mainly explores the efficient parameter fine-tuning of pre-trained models in the downstream tasks. Therefore, our experiments do not involve the loss of CD and EMD, and the loss in our experiments is the loss function of various downstream tasks, such as cross-entropy loss for classification.

**Comment by gyjung975** [comment]
Thanks for the reply! I will close the issue.
